packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -1,27 +1,186 @@
+      // [HighContrastSelector]: {
+      //   forcedColorAdjust: 'none',
+      // },
+    },
+
+    ':active .ms-Slider-thumb::after': {
+      background: '#005A9E',
+      // [HighContrastSelector]: {
+      //   forcedColorAdjust: 'none',
+      // },
+    },
+  }),
+});
+
+/**
+ * Styles for the rail slot
+ */
+const useRailStyles = makeStyles({
+  rail: theme => ({
+    position: 'absolute',
+    height: '4px',
+    width: '100%',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    background: '#C8C8C8',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+    // [HighContrastSelector]: {
+    //   boxSizing: 'border-box',
+    //   border: '1px solid WindowText',
+    // },
+  }),
+});
+
+/**
+ * Styles for the track slot
+ */
+const useTrackStyles = makeStyles({
+  track: theme => ({
+    position: 'absolute',
+    height: '4px',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    minWidth: 'var(--slider-thumb-size)',
+    background: '#606060',
+    borderRadius: '4px',
+    // [HighContrastSelector]: {
+    //   background: 'Highlight',
+    // },
+  }),
+});
+
+/**
+ * Styles for the thumb slot
+ */
+const useThumbStyles = makeStyles({
+  thumb: theme => ({
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    left: 'calc(var(--slider-thumb-size) / 2)',
+    right: 'calc(var(--slider-thumb-size) / 2)',
+    outline: 'none',
+    ':after': {
+      content: '""',
+      position: 'absolute',
+      width: 'var(--slider-thumb-size)',
+      height: 'var(--slider-thumb-size)',
+      top: '50%',
+      background: '#606060',
+      borderRadius: '50%',
+      boxSizing: 'border-box',
+      display: 'block',
+      transform: 'translate(-50%,-50%)',
+      // [HighContrastSelector]: {
+      //   background: 'Highlight',
+      // },
+    },
+  }),
+
+  focusIndicator: createFocusIndicatorStyleRule({
+    ':after': {
+      outline: 'none',
+      boxSizing: 'border-box',
+      border: '1.7px solid black',
+      boxShadow: '0 0 0 .7pt white inset',
+      // [HighContrastSelector]: {
+      //   background: 'GrayText',
+      //   border: '1.5px solid WindowText inset',
+      // },
+    },
+  }),
+});
+
+/**
+ * Styles for the activeRail slot
+ */
+const useActiveRailStyles = makeStyles({
+  activeRail: theme => ({
+    position: 'absolute',
+    left: 'calc(var(--slider-thumb-size) / 2)',
+    right: 'calc(var(--slider-thumb-size) / 2)',
+  }),
 });
 
 /**
  * Apply styling to the Slider slots based on the state
  */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const useSliderStyles = (state: any): any => {
-  const styles = useStyles();
-  state.className = mergeClasses(styles.root, state.className);
+export const useSliderStyles = (state: SliderState): SliderState => {
+  const rootStyles = useRootStyles();
+  const railStyles = useRailStyles();
+  const activeRailStyles = useActiveRailStyles();
+  const trackStyles = useTrackStyles();
+  const thumbStyles = useThumbStyles();
```

Comment:
_Just for curiosity_: what is a reason to have multiple `makeStyles()` call?

```tsx
const styles = makeStyles({
  root: {},
  label: {},
  inputChecked: {},
})

// vs

const styles = makeStyles({ root: {} })
const labelStyles = makeStyles({ root: {} })
const inputStyles = makeStyles({ checked: {} })
```

Reply:
I felt that it was more readable as it helps separate each of the slots styles. Are there downsides to having multiple calls?